osu.Game.Rulesets.Osu/Skinning/Default/SpinnerSpmCounter.cs

```diff
@@ -21,6 +21,11 @@ public class SpinnerSpmCounter : Container
 
         private readonly OsuSpriteText spmText;
 
+        public override void ApplyTransformsAt(double time, bool propagateChildren = false)
+        {
+            // handles own fade in state.
```

Comment:
(Forgot to save the comment before posting. Oops.)

Not sure about this to be fair. I'm thinking we will want to have the spin start time in the judgement at some point (for stats), and using that would allow doing this with a normal absolute sequence, right? I think I'd prefer that. We have another place where we block transform application, but it's arguably quite special (it's `DrawableHitObject` itself).

Reply:
No worries, I'll give that path a try.

Reply:
Seems to work quite well, and read better